Makefile

```diff
@@ -116,6 +122,9 @@ clean-hub: ensure-operator-sdk
 cluster-ip:
   CLUSTER_IP?=$(shell $(KUBECTL) get svc kubernetes -n default -o jsonpath="{.spec.clusterIP}")
 
+cluster-ip-kind:
+  CLUSTER_IP_KIND?=$(shell $(KUBECTL) --kubeconfig=$(HUB_KIND_KUBECONFIG) config view | grep server | awk '{ print $2 }' | cut -f3 -d/ | cut -f1 -d:)
```

Comment:
it seems to me you do not need HUB_KIND_KUBECONFIG, KUBECONFIG is good enough?

Reply:
In my PR for the website, I actually changed exporting of the kubeconfig from hub to the managed one see: https://github.com/mikeshng/website/blob/update-foundation/content/getting-started/register-cluster.md#install-from-source

The main reason I did this is because of the olm install step doesn't seem to take in a kubeconfig and just uses whatever kubeconfig env is currently set. 
`$(OPERATOR_SDK) olm install --version $(OLM_VERSION)`
and during the deploy spoke we want to install the olm in the managed cluster.

So technically I actually didn't need to define `KLUSTERLET_KIND_KUBECONFIG` anymore. 

Reasons why I have both `HUB_KIND_KUBECONFIG` and `KLUSTERLET_KIND_KUBECONFIG` defined:
- the deploy spoke step is very complicated because involves interacting with the hub and the managed kubeconfig. having a clear variable makes the code a bit easier to understand.
- the `HUB_KIND_KUBECONFIG` is needed for the hub secret creation step
- I changed the instruction on the website to export KUBECONFIG as the managed one also makes it a bit easier to follow for the user since they are now prepping the managed cluster so it's more natural to export the kubeconfig as the managed one.